rust/hyperlane-base/src/types/local_storage.rs

```diff
@@ -62,6 +62,14 @@ impl CheckpointSyncer for LocalStorage {
         }
     }
 
+    async fn write_latest_index(&self, index: u32) -> Result<()> {
+        let path = self.latest_index_file_path();
+        tokio::fs::write(&path, index.to_string())
+            .await
+            .with_context(|| format!("Writing index to {path:?}"))?;
+        Ok(())
+    }
+
```

Comment:
I have no idea how this happened or we didn't see earlier but check out https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/latest-index-test/rust/hyperlane-base/src/types/local_storage.rs#L36-L39, this should be the same as the S3 one I think? https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/b141aece73e6ec910c6d2d3294edad008d6db3df/rust/hyperlane-base/src/types/s3_storage.rs#L134-L136

Maybe this is why e2e is failing?

Reply:
Can we actually just put these strings into shared consts

Reply:
actually hmm I don't think this would cause things to fail anyways

Reply:
these don't need to be the same for the different modalities right? agree they probably should be shared

Reply:
yeah you're right, they don't need to be the same, my mistake